File: InMotionGIT.Common/Services/Contracts/Credential.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract()>
    Public Class Credential

        <DataMember()>
        Public Property ConnectionStringName As String

        <DataMember()>
        Public Property User As String

        <DataMember()>
        Public Property Password As String

    End Class

End Namespace

File: Common/InMotionGIT.Common/Services/Contracts/Credential.cs
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Services.Contracts
{

    [DataContract()]
    public class Credential
    {

        [DataMember()]
        public string ConnectionStringName { get; set; }

        [DataMember()]
        public string User { get; set; }

        [DataMember()]
        public string Password { get; set; }

    }

}